cfme/containers/provider/__init__.py

```diff
@@ -228,18 +228,18 @@ def num_project(self):
     @num_project.variant('ui')
     def num_project_ui(self):
         view = navigate_to(self, "Details")
-        return int(view.entities.summary("Relationships").get_text_of("Projects"))
+        name = VersionPick({Version.lowest(): 'Projects',
+                            '5.9': 'Container Projects'})
```

Comment:
**REQUIRED**
Please just replace`Projects` be `Container Projects`. CFME 5.8 is not supported by the master.